### ISSUE TITLE
Yarn can't install public github repos

### ISSUE THREAD

**Issue by georgediaz88** [heroku/heroku-buildpack-nodejs#361]
I'm making the switch from Npm to Yarn on Heroku and using the `#yarn` branch of this buildpack and running into an installation problem. When trying to install a public _github_ package through Yarn, the install hangs and timeouts on Heroku at step [2/4]:

<img width="1186" alt="screenshot 2017-01-05 20 11 02" src="https://cloud.githubusercontent.com/assets/667650/21705120/80da3c6a-d383-11e6-8f79-27ab1cef174b.png">

for package:
`"jspm": "https://github.com/jspm/jspm-cli.git#some-sha-here"`

I'm pretty sure it's the "jspm" github package im pointing to in my _package.json_ file because when I remove that line the install works.  The frustrating part is everything installs fine locally. So, I'm wondering if this is a current issue with installing a github Yarn packages on Heroku? Any help would be appreciated. Thanks.

**Comment by georgediaz88** [comment]
Hello. Any update on this?

**Comment by hunterloftis** [comment]
Yarn support is in master (there's no need to use any branch, and the branches will be out of date). What happens when you push with the default buildpack?

**Comment by georgediaz88** [comment]
@hunterloftis , thanks for responding. Cool, I didn't realize it was in "master" and going through some older issues on this repo, I saw a lot of recommendations point the buildpack to the"#yarn" branch. Anyway, I switched back to using that buildpack's "master" branch and manually installed "jspm" using the github branch and saw it working following the Yarn docs and see its now working! 

I just update the yarn.lock file locally and pushed that as well to see it still works which is great. 

Cool thanks for the suggestion to use the "master" branch. I think that helped! I'll close this issue. Have a good day!
